lms/djangoapps/teams/static/teams/js/views/team_card.js

```diff
@@ -72,30 +75,30 @@
         TeamActivityView = Backbone.View.extend({
             tagName: 'div',
             className: 'team-activity',
-            template: _.template(dateTemplate),
+            template: HtmlUtils.template(dateTemplate),
 
-            initialize: function (options) {
+            initialize: function(options) {
                 this.date = options.date;
             },
 
-            render: function () {
+            render: function() {
                 var lastActivity = moment(this.date),
                     currentLanguage = $('html').attr('lang');
                 lastActivity.locale(currentLanguage);
                 this.$el.html(
-                    interpolate(
-                        // Translators: 'date' is a placeholder for a fuzzy, relative timestamp (see: http://momentjs.com/)
-                        gettext("Last activity %(date)s"),
-                        {date: this.template({date: lastActivity.format('MMMM Do YYYY, h:mm:ss a')})},
-                        true
-                    )
+                    HtmlUtils.interpolateHtml(
+                        // Translators: 'date' is a placeholder for a fuzzy, relative timestamp
+                        // (see: http://momentjs.com/)
+                        gettext('Last activity {date}'),
+                        {date: HtmlUtils.HTML(this.template({date: lastActivity.format('MMMM Do YYYY, h:mm:ss a')}))}
```

Comment:
This is the first time I hit something from our discussion the other day.  It struck me as odd looking, but I think it is good as is.